.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug-report
+about: Create a report to help us improve
+title: "[Bug]"
+labels: bug
+assignees: ''
+
+---
+
+## Summary of the Problem
+_Please provide two-to-three sentences of the issue(s) you experienced._ 
+
+
+### Reproducing the Problem
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Calendar Data Collecting
+
+__What Calendar system were you using?__ 
+*Harptos, Earth, Etc* 
+
+__Was it created within the plugin, or imported from Fantasy Calendars online?__
```

Comment:
Just a note: The web app's name is Fantasy Calendar, singular. 

Do that, and I think this is fine.

Reply:
Should be updated now. :)